### DISCUSSION TITLE
Adaptation to the Notch of smartphones

### DISCUSSION THREAD

**Discussion by ramon-rodrigues3** [godotengine/godot-proposals#6380] [Platforms]
It would be very useful if there was a native option in the editor that ignored the smartphone notch when stretching the screen.

**Comment by dukemagus** [comment]
i have no idea how to make this setting in the preview, but for android 9 onwards, the API already has it ready:

https://developer.android.com/develop/ui/views/layout/display-cutout
https://developer.android.com/reference/android/view/WindowManager.LayoutParams#LAYOUT_IN_DISPLAY_CUTOUT_MODE_NEVER

**Reply by ramon-rodrigues3** [reply]
Does this mean that if the min sdk is Android version 9 when the game is stretched to occupy the entire screen, it will respect the notch area?
